src/test/util/exec.ts

```diff
@@ -0,0 +1,58 @@
+
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {ExecOptions} from 'child_process';
+import util = require('util');
+const {promisify} = util;
+const {execFile: _execFile} = require('child_process');
+const execFile = promisify(_execFile);
+
+export interface ExecResult {
+  stdout: string;
+  stderr: string;
+}
+
+/**
+ * A helper function for working with Node's core execFile() method.
+ */
+export default async function exec(
```

Comment:
@FredKSchott Should I move these into the `src/util.ts` file instead of under test utils?

Reply:
👍 that makes sense, I'll make sure I get that upstream in my PR